Cargo.toml

```diff
@@ -42,7 +42,7 @@ log = "0.4.25"
 nix = "0.29"
 openssl = "0.10"
 prost = "0.13"
-protobuf = "3.5.1"
+protobuf = "=3.7.1"
```

Comment:
I think we might not need the `=` here to fix the version?

Reply:
It's not required to fix things today, but I am thinking we should pin the version so that we don't run into this problem again. Otherwise the build will break again next time there is an update. Hopefully depandabot will tell us when a new version is available and we can update manually.

In the future we might want to change how the image_pull protos are built to resolve this